cardano-api/gen/Test/Gen/Cardano/Api/Typed.hs

```diff
@@ -512,7 +512,7 @@ genReferenceScript :: CardanoEra era -> Gen (ReferenceScript era)
 genReferenceScript era =
   case refInsScriptsAndInlineDatsSupportedInEra era of
     Nothing -> return ReferenceScriptNone
-    Just supp -> ReferenceScript supp <$> genScriptInAnyLang
+    Just _ -> scriptInEraToRefScript <$> (genScriptInEra era)
```

Comment:
This was needed so that we did not generate Plutus V3 scripts in the babbage era. But I had to export `scriptInEraToRefScript` from `Cardano/Api/Script`.